circuits/blindbid/src/error.rs

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+//! Error definitions for the BlindBid circuit & gadgets.
+
+use core::fmt;
+use dusk_blindbid::BlindBidError;
+use dusk_bytes::Error as DuskBytesError;
+use dusk_plonk::error::Error as PlonkError;
+use dusk_poseidon::Error as PoseidonError;
+
+#[derive(Debug)]
+/// Compilation of the Error definitions for the BlindBid circuit & gadgets.
+pub enum BlindBidCircuitError {
```

Comment:
If we are `std` then we should probably implement `std::error::Error` for this type with some additional `PartialEq` to facilitate error matching